packages/trader/src/App/Containers/trade-footer-extensions.tsx

```diff
@@ -40,8 +39,4 @@ const TradeFooterExtensions = observer(() => {
     return null;
 });
 
-TradeFooterExtensions.propTypes = {
```

Comment:
`location` was defined in proptypes but wasn't passed as a prop

Reply:
@henry-deriv `withRouter`, which TradeFooterExtensions is wrapped into, is providing this component with `location`. It was received via props (as it should) before it was accidentally removed from props in [this PR](https://github.com/binary-com/deriv-app/pull/8270/files#diff-76bac00b8e4f3974736ca37355f517529bc2c84bb9b3d83297e5eefd467bca3d). Please return it to props. The type for props in this case will be `RouteComponentProps<{}>` (empty object is a replacement for the component props type because there are no additional props received via props).
`RouteComponentProps` will be imported from:
```
import { RouteComponentProps } from "react-router";
```

Reply:
done. thanks for clarification